src/i18n.ts

```diff
@@ -267,6 +268,29 @@ function validatePath(path: string): void {
 }
 
 /**
+ * Return the cached messages for the specified bundle and locale. If messages have not been previously loaded for the
+ * specified locale, no value will be returned.
+ *
+ * @param bundle
+ * The bundle object or path.
+ *
+ * @param locale
+ * The locale of the desired messages.
+ *
+ * @return The cached messages object, if it exists.
+ */
+export function getCachedMessages<T extends Messages>(bundlePath: string, locale: string): T | void;
+export function getCachedMessages<T extends Messages>(bundle: Bundle<T>, locale: string): T | void;
+export function getCachedMessages<T extends Messages>(bundle: any, locale: string): T | void {
+	const bundlePath = (typeof bundle === 'string') ? bundle : bundle.bundlePath;
+	const cached = bundleMap[bundlePath];
+
+	if (cached) {
+		return cached[locale] as T;
```

Comment:
Without the `as T`, the TS compiler warns that `Messages` cannot be assigned to type `void | T`; hence the casting.